File: 04_time_and_types/02/main.go
package main

import (
	"fmt"
)

// Write a program that prints a number in decimal, binary, and hex.
func prints_values(input_from_outside int) {
	var input int
	fmt.Print("Enter a number: ")
	fmt.Scan(&input)

	fmt.Printf("Decimal: %d\n", input)
	fmt.Printf("Binary: %b\n", input)
	fmt.Printf("Hexadecimal: %#X\n", input) // that will print the input number with a leading '0x' or '0X' depends on the case
}

File: 04_time_and_types/02/main.py
def main(input_value_from_outside=None):
    if input_value_from_outside is None:
        input_value = int(input("Enter a number: "))
    else:
        input_value = input_value_from_outside

    print("Decimal: ", input_value)
    print("Binary: ", bin(input_value))
    print("Hexadecimal: ", hex(input_value))
    return "Decimal: " + str(input_value) + " Binary: " + bin(input_value) + " Hexadecimal: " + hex(input_value) 

if __name__ == "__main__":
    main()
